packages/protocol/contracts/nft_bridge/erc721/IErc721Bridge.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: MIT
+//  _____     _ _         _         _
+// |_   _|_ _(_) |_____  | |   __ _| |__ ___
+//   | |/ _` | | / / _ \ | |__/ _` | '_ (_-<
+//   |_|\__,_|_|_\_\___/ |____\__,_|_.__/__/
+
+pragma solidity ^0.8.20;
+
+/**
+ * Bridge interface for NFT contracts.
+ * @dev Ether is held by Bridges on L1 and by the EtherVault on L2,
+ * not TokenVaults.
+ */
+interface IErc721Bridge {
+    struct Message {
```

Comment:
In Bridge.sol, the Message is generic, it's not designed for ERC20, and we shall not design a special Message for NFTs either.